module/VuFind/src/VuFind/Export.php

```diff
@@ -107,6 +107,10 @@ public function getRedirectUrl(string $format, string $callback): string
         foreach ($matches[1] as $current) {
             $parts = explode('|', $current);
             switch ($parts[0]) {
+                case 'route':
+                    $urlHelper = $this->viewRenderer->plugin('url');
+                    $template = str_replace('{' . $current . '}', $urlHelper($parts[1] ?? '??'), $template);
```

Comment:
Is `'??'` really intended here? I assume that's just going to throw an error about an undefined route.